sinks/api/supported_labels.go

```diff
@@ -15,13 +15,15 @@
 package api
 
 const (
-	LabelPodId         = "pod_id"
-	LabelPodName       = "pod_name"
-	LabelPodNamespace  = "pod_namespace"
-	LabelContainerName = "container_name"
-	LabelLabels        = "labels"
-	LabelHostname      = "hostname"
-	LabelResourceID    = "resource_id"
+	LabelPodId               = "pod_id"
+	LabelPodName             = "pod_name"
+	LabelPodNamespace        = "pod_namespace"
+	LabelContainerName       = "container_name"
+	LabelLabels              = "labels"
+	LabelHostname            = "hostname"
+	LabelResourceID          = "resource_id"
+	LabelComputeResourceID   = "compute.googleapis.com/resource_id"
```

Comment:
Nope. According to https://cloud.google.com/compute/docs/autoscaler/scaling-cloud-monitoring-metrics#custom_metrics_beta it has to be exactly:
compute.googleapis.com/resource_type
compute.googleapis.com/resource_id